server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -48,6 +48,12 @@ DynamicFieldsBuilder getDynamicFieldsBuilder() {
                 return DynamicFieldsBuilder.DYNAMIC_TRUE;
             }
         },
+        UNTIL_LIMIT {
```

Comment:
I would maybe call this TRUE_UNTIL_LIMIT just to clarify that it maps indexed fields until limit, otherwise one may wonder where this new dynamic mode adds the new fields (runtime or properties?)

Reply:
Done